pom.xml

```diff
@@ -7,7 +7,7 @@
 	<artifactId>opensrp-server-common</artifactId>
 	<packaging>jar</packaging>
 
-	<version>1.0.0-SNAPSHOT</version>
+	<version>1.0.2-SNAPSHOT</version>
```

Comment:
Any reason this is skipping `1.0.1-SNAPSHOT`?

Reply:
oops 😄 downgrading version...